README.md

```diff
@@ -171,4 +171,4 @@ Otherwise, not all sequences are updated with the correct extent and the bundled
 ## LottieMorphs
 
 If you are in posession of the complete LottieMorph source code, you need to copy-paste the complete implementation into `interactive-morphs/lottie-morph.js`.
-No other steps are required and you can start creating fully functioning LottieMorphs e.g., via the TopBar.
\ No newline at end of file
+No other steps are required and you can start creating fully functioning LottieMorphs e.g., via the TopBar.
```

Comment:
I am not sure what this is - different line ending character?

Reply:
I don't know, meld does not show anything